### PR TITLE
Changed walkthrough from UPS to mini

### LINKED ISSUE #2658: Replace use of UPS broker in docs walkthrough with minibroker
Currently, we make extensive use of the [ups broker](https://github.com/kubernetes-sigs/service-catalog/tree/master/charts/ups-broker) in our documentation. The UPS broker is a dummy-broker that doesn't actually do anything, and it's name is often confusing for users.

Replace all use of the ups broker with [minibroker](https://github.com/osbkit/minibroker) a broker that deploys helm charts for various databases as OSB services. This is a much better example broker to use as it deploys actual services.

Known things that need to be rewritten:
[The walkthrough on the docsite](https://github.com/kubernetes-sigs/service-catalog/blob/master/docs/walkthrough.md) - this is the basic walkthrough of installing a broker, provisioning a service, and binding that service. Now might also be a good time to replace all the use of raw kubectl with svcat.
[the resources used by the wallthrough](https://github.com/kubernetes-sigs/service-catalog/tree/master/contrib/examples/walkthrough) - this is all the yaml files used by the walkthrough kubectl commands. These may go away entirely if we switch to svcat.

### PR DESCRIPTION
**Pull request by zulandar**
This PR is a 
 - [ ] Feature Implementation
 - [ ] Bug Fix
 - [X] Documentation

**What this PR does / why we need it**:
Requested change from UPS Broker to Minibroker within walkthrough.

**Which issue(s) this PR fixes** 
Fixes #2658

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -13,159 +13,119 @@ of this repository.
...
-In this repository, there's a simple, "dummy" server called the User Provided
-Service (UPS) broker. The codebase for that broker is
-[here](https://github.com/kubernetes-sigs/service-catalog/tree/master/contrib/pkg/broker/user_provided/controller).
+We plan on using the minibroker for demo purposes. The codebase for that broker is
+[here](https://github.com/osbkit/minibroker).
 
-We're going to deploy the UPS broker to our Kubernetes cluster before
-proceeding, and we'll do so with the UPS helm chart. You can find details about
+We're going to deploy the minibroker to our Kubernetes cluster before
+proceeding, and we'll do so with the minibroker helm chart. You can find details about
```
Comment (jberkhahn) [inline docs/walkthrough.md]: should be "you can find details about the chart in the minibroker README"

File: docs/walkthrough.md [hunk 2]
```diff
@@ -13,159 +13,119 @@ of this repository.
...
+                     mass-deployed software.         
+  10-1-28            Fast, reliable, scalable,       
+                     and easy to use open-source     
+                     relational database system.     
+                     MariaDB Server is intended      
+                     for mission-critical,           
+                     heavy-load production systems   
+                     as well as for embedding into   
+                     mass-deployed software. 
+
```
Comment (jberkhahn) [inline docs/walkthrough.md]: maybe add a 
.
.
.
here to indicate the list of plans has been truncated?

File: docs/walkthrough.md [hunk 3]
```diff
@@ -176,67 +136,85 @@ resources available in the cluster:
...
+                                              as well as for embedding into    
+                                              mass-deployed software.          
+  10-1-28                        mariadb      Fast, reliable, scalable,        
+                                              and easy to use open-source      
+                                              relational database system.      
+                                              MariaDB Server is intended       
+                                              for mission-critical,            
+                                              heavy-load production systems    
+                                              as well as for embedding into    
+                                              mass-deployed software.
```
Comment (jberkhahn) [inline docs/walkthrough.md]: same thing here, add a vertical ellipsis to show its been truncated

File: docs/walkthrough.md [hunk 4]
```diff
@@ -176,67 +136,85 @@ resources available in the cluster:
...
+  externalName: 10-1-26
   free: true
 status:
   removedFromBrokerCatalog: false
 ```
 
 # Step 4 - Creating a New ServiceInstance
 
-Now that a `ClusterServiceClass` named `user-provided-service` exists within our
+Now that a `ClusterServiceClass` named `10-1-26` exists within our
```
Comment (jberkhahn) [inline docs/walkthrough.md]: this should be `a 'ClusterServiceClass named 'mariadb' exists`, and all the references should be that

File: docs/walkthrough.md [hunk 5]
```diff
@@ -414,37 +398,37 @@ see a new one:
...
-NAME                  TYPE                                  DATA   AGE
-default-token-v24x9   kubernetes.io/service-account-token   3      1m
-ups-binding           Opaque                                2      37s
+NAME                    TYPE                                  DATA   AGE
+default-token-n2j75     kubernetes.io/service-account-token   3      10m
+mini-binding            Opaque                                8      91s
 ```
 
-Notice that a new `Secret` named `ups-binding` has been created.
+Notice that a new `Secret` named `mini-binding` has been created.
```
Comment (jberkhahn) [inline docs/walkthrough.md]: could you add a blurb here like "At this point,we could use this secret to connect the running MariaDB database to our application running on Kubernetes."

**Comment by jberkhahn** [comment]
mentioned a couple things, other than that this looks good

**Comment by zulandar** [comment]
> mentioned a couple things, other than that this looks good

Made the recommended adjustments.

**Comment by zulandar** [comment]
/retest

No code changes was performed to cause test failures. Here is the failure from the test.

```
Setting up ca-certificates (20161130+nmu1+deb9u1) ...
Updating certificates in /etc/ssl/certs...
/usr/sbin/update-ca-certificates: 101: /usr/sbin/update-ca-certificates: sed: Exec format error
/usr/sbin/update-ca-certificates: 75: /usr/sbin/update-ca-certificates: rm: Exec format error
dpkg: error processing package ca-certificates (--configure):
 subprocess installed post-installation script returned error exit status 2
Processing triggers for libc-bin (2.24-11+deb9u4) ...
Errors were encountered while processing:
 ca-certificates
[91mE[0m[91m: Sub-process /usr/bin/dpkg returned an error code (1)
[0mThe command '/bin/sh -c export DEBIAN_FRONTEND=noninteractive &&     apt-get update &&     apt-get install ca-certificates -y &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
Makefile:383: recipe for target 'service-catalog-image' failed
make[1]: *** [service-catalog-image] Error 100
make[1]: Leaving directory '/home/prow/go/src/github.com/kubernetes-sigs/service-catalog'
Makefile:348: recipe for target 'arch-image-arm64' failed
make: *** [arch-image-arm64] Error 2
+ EXIT_VALUE=2
+ set +o xtrace
```

**Comment by mszostok** [comment]
@zulandar currently is a known issue that the pull-service-catalog-xbuild is sometimes failing with such `/usr/sbin/update-ca-certificates: rm: Exec format error` error you need to add comment 

```
/test pull-service-catalog-xbuild
```

to execute that test one more time.

**Pull request merged**